app/models/role.rb

```diff
@@ -30,8 +30,14 @@ class Role < ApplicationRecord
 
   scopify
 
+  def self.ransackable_attributes(auth_object = nil)
+    %w[name]
+  end
+
   # Removes all users from the role.
   def purge
+    return if name.downcase.strip == 'Member'
```

Comment:
I would think `.downcase` would make the M lower case?
```suggestion
    return if name.downcase.strip == 'member'
```